pkg/edit/editor_test.go

```diff
@@ -37,6 +37,36 @@ func TestEditor_DoesCommandWithLeadingSpaceToHistory(t *testing.T) {
 	testCommands(t, f.Store /* no commands */)
 }
 
+func TestEditor_TestBeforeAddCmd(t *testing.T) {
+	cases := []struct {
+		name string
+		rc   string
+		in   string
+		exp  []string
```

Comment:
Follow the naming convention of other test code, and call this field `wantHistory`.